test/conformance/conformance_test.go

```diff
@@ -52,7 +52,46 @@ func TestBrokerConformance(t *testing.T) {
 	env.Prerequisite(ctx, t, secret.Created("rabbitbroker-secret", "rabbitbroker", ctx, b.WithEnvConfig()...))
 	// Install and wait for a Ready Broker.
 	env.Prerequisite(ctx, t, broker.GoesReady("default", b.WithEnvConfig()...))
+	brokerFeatureSet := broker.ControlPlaneConformance("default")
+	// To distribute the test total load, 8 gave me the best results locally
+	brokerFeatureSet.Features = brokerFeatureSet.Features[:8]
+	env.TestSet(ctx, t, brokerFeatureSet)
+}
+
+func TestBrokerControlPlaneConformance2(t *testing.T) {
```

Comment:
I understand setting `-parallel=8` to lighten the load but I'm confused about splitting up the Conformance tests by feature here. I don't know much abt the reconciler-test framework that sets up the features, but I naively thought setting the parallel flag would be enough?

Reply:
I tested locally and without the split up the errors appeared

Reply:
What I think can be happening, is that the env of the feature set is not been cleaned up  when its in the same test suite. But splitting test suites the feature set env get cleaned up more times

Reply:
Yeah it looks like it would recreate the env if split into different tests. From what I can tell, the env is pretty much a RabbitMQ cluster and a default broker? I don't think the default broker gets used, it feels more like a way to get a ready signal before kicking off the tests. The tests seem to create brokers for themselves

Reply:
Yep, that's what the basic tests create, buuut. In the first test the broker is just been tested for the ready condition, in the other tests sometimes triggers are being created for the default broker and sometimes for a new broker. Is pretty weird. To give more context, there are a total of 18 features that are being tested

Reply:
Ah yes I see it, thanks. Some seem to use the randomized name and then some use the broker given to them.

Reply:
It's getting better at least xD just one Red workflow!